changelog/changelog.ddoc

```diff
@@ -15,14 +15,15 @@ SUBNAV=$(SUBNAV_TEMPLATE
 SEARCH_OPTIONS_EXTRA=<option value="dlang.org/changelog" selected>Change
 Log</option>
 
-VERSION=
+STABLE_VERSION=
```

Comment:
To be fair: it wouldn't be too hard to continue to generate `VERSION` at the `changed.d` script, but imho this helps clarifying things.